src/app/services/services.module.ts

```diff
@@ -3,7 +3,6 @@ import { ToastrModule } from "ngx-toastr";
 
 import {
   JslibServicesModule,
-  WINDOW,
```

Comment:
Why are we removing this? Doesn't seem related to the proposed changes.

Reply:
It's an unused import - just cleaning it up while I'm here.